packages/flutter/test/gestures/mouse_tracking_test.dart

```diff
@@ -93,7 +93,6 @@ void main() {
       _pointerData(PointerChange.hover, const Offset(1.0, 401.0), device: 1),
     ]);
     RendererBinding.instance.mouseTracker.attachAnnotation(annotation);
-    RendererBinding.instance.mouseTracker.sendMouseNotifications(<int>{0});
```

Comment:
Why do we no longer need to send sendMouseNotifications in the test?

Reply:
The tests need something to trigger the notification. The earliest implementation of tests had to use this private API, but we have since moved to public API (`attachAnnotation` and `onPointerDataPacket`), which will call `sendMouseNotifications` as they should in real apps. Calling a `sendMouseNotifications` again is not only redundant, but wrong, as [one test](https://github.com/flutter/flutter/pull/41879/files/aaed56be757be097601cc0f7ec77fc93c180b2ef#diff-e905ee2a85cdfc5131bead06d39d8cd3L131) has to be fixed for having an extra event.